static/css/add.scss

```diff
@@ -18,14 +18,24 @@
   // Boundary box around form
   &__box {
     display: block;
-    width: 32.5em;
+    width: $box-width;
     border: 0.05em solid $main;
     border-radius: 0.5em;
     padding: 1em 1.5em;
     margin-right: auto;
     margin-left: auto;
     margin-bottom: 1em;
     box-shadow: 0.1em 0.1em 0.25em $secondary;
+    
+    &--buttons {
```

Comment:
Why is this a modifier (`--`)? Should probably be an element (`__`) (double underscore).